public/app/features/alerting/unified/components/rules/NoRulesCTA.tsx

```diff
@@ -19,6 +21,7 @@ export const NoRulesSplash: FC = () => {
         proTipLink="https://grafana.com/docs/"
         proTipLinkTitle="Learn more"
         proTipTarget="_blank"
+        onClick={() => logInfo(LogMessages.alertRuleFromScratch)}
```

Comment:
Shouldn't we add the same to the `+ New alert rule` button?
![image](https://user-images.githubusercontent.com/6293563/192236788-2191a574-1701-47aa-b2ac-7f8a2338ce06.png)

Reply:
I think it's this one, right? https://github.com/grafana/grafana/blob/54904-js-agent-trackings/public/app/features/alerting/unified/RuleList.tsx#L106
If so, it already has that tracking.